### PR TITLE
Issues/swodlr UI 72 essential UI bug fixes

### PR DESCRIPTION
**Pull request by jbyrne6**
### Changes
In this PR are bug fixes for the following bug fixes. The corresponding github issue for this PR is #72.

1. When generating products using map spatial search option (BBOX), various warnings display while scenes are being retrieved / added (“The scenes entered are not available.” “Some scenes entered are not available.“)
2. (2) ‘Configure Products’ should not be an option after adding scenes in UI, as this configuration already occurred via Select Scene ‘Configure Options’ (top of the Customization page in UI)
3. Customization page in UI - user unable to delete Added Scenes. When refreshing the page after added scenes removed/deleted, the deleted scenes re-appear in SWODLR UI FIX: remove url param when deleting from table
4. Customization page in UI - Map sizing issues. The size of map should remain in place. The Map in UI should not randomly re-size (e.g., after sizing, and navigating through menus, and back to customization page w/ map or when refreshing the page)
5. ‘My Data’ page in UI: (user only can only view / display 10 rows of generated data in UI) Generated Products Data do not retain in UI (Should pagination or page numbers be Implemented ?)
6. ‘Configure Options / Products’ page - advanced options ‘Scenes to Customize’ - User unable to toggle or select all of the row options for UTM Zone Adjust or MGRS Band Adjust
7. Tutorial modal does now start when you click the **Tutorial** navbar button when on certain pages.

### CODE REVIEW

File: src/components/app/App.tsx [hunk 1]
```diff
@@ -34,20 +35,21 @@ const App = () => {
...
     const { action, step, type, lifecycle } = data;
     const stepTarget = step.target
     if (stepTarget === '#configure-options-breadcrumb' && action === 'update') {
       navigate(`/customizeProduct/configureOptions${search}`)
-    } else if (stepTarget === '#configure-options-breadcrumb' && action === 'prev') {
+    } else if (stepTarget === '#configure-options-breadcrumb' && action === 'prev' && lifecycle === 'complete') {
       navigate(`/customizeProduct/selectScenes${search}`)
-    } else if (stepTarget === '#my-data-page' && action === 'prev') {
+    } else if (stepTarget === '#alert-messages' && action === 'prev' && lifecycle === 'init') {
+      // navigate(`/customizeProduct/selectScenes${search}`)
```
Comment (frankinspace) [inline src/components/app/App.tsx:50]: Empty conditional? If this is intentional can you put a comment as to why it is empty

File: src/components/sidebar/GranuleSelectionView.tsx [hunk 2]
```diff
@@ -1,28 +1,20 @@
...
-    const { search } = useLocation();
-
   return (
-    <>
+    <div>
         <SpatialSearchOptions />
         <GranuleTable tableType='granuleSelection'/>
         <GranuleTableAlerts />
-        <hr></hr>
+        {/* <hr></hr>
```
Comment (frankinspace) [inline src/components/sidebar/GranuleSelectionView.tsx:11]: can we remove instead of comment?

File: src/components/tutorial/tutorialConstants.ts [hunk 3]
```diff
@@ -116,17 +116,17 @@ export const tutorialSteps = [
...
-      content: "Click the Configure Products button to proceed to the Configure Options page where you can select options for how your selected scenes can be made into custom products.",
-      disableBeacon: true,
-      styles: {
-          options: {
-              zIndex: 1000,
-              primaryColor: '#0d6efd',
-          }
-      }
-    },
+    // {
```
Comment (frankinspace) [inline src/components/tutorial/tutorialConstants.ts:119]: remove

File: src/components/sidebar/GranuleSelectionView.tsx (hunk 2 above)
Comment (jbyrne6) [reply src/components/sidebar/GranuleSelectionView.tsx:11]: Yes, I will clean up comments

**Pull request merged**